core/primitives/src/version.rs

```diff
@@ -128,6 +128,9 @@ pub enum ProtocolFeature {
     /// Make block producers produce chunks for the same block they would later produce to avoid
     /// network delays
     SynchronizeBlockChunkProduction,
+    /// Change the algorithm to count WASM stack usage to avoid undercounting in
+    /// some cases.
+    CorrectStackLimit,
```

Comment:
if we ever need to correct the algorithm again, it'll be hard to come up with a name for it ^^

Reply:
This name we can retroactively change in the future, as it is not serialized anywhere, so tried not to over-thing this )